quantum/nodes/grovers-algo/grovers-algo.html

```diff
@@ -0,0 +1,28 @@
+<script type="text/javascript">
+  RED.nodes.registerType("grovers-algo", {
+    category: "quantum",
+    color: "#8a3ffc",
```

Comment:
I think we should change the colour to something else, since purple is used for certain gate nodes currently. @Theo-Reignier any suggestions which colour you think would be best?

Reply:
I would like to give it the `gray` color since it is a classical node (classical input, classical output, all quantum is encapsulated). However, we really need to differentiate it from the 'algorithm control' grey nodes.

What about a darker shade of grey?

Reply:
I agree, let's go with this shade for now #605e64

Reply:
This looks good to me, just ensure the text is properly readable on this background please !

Reply:
Rather this is one is more clearer #737177